.github/workflows/go.yml

```diff
@@ -0,0 +1,102 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.12', '1.13', '1.14']
+
+    steps:
+      - name: Install protobuf
+        uses: arduino/setup-protoc@v1.1.0
+        with:
+          version: '3.x'
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+        id: go
+
+      - name: Install protoc-gen-go
+        run: |
+          go get -u google.golang.org/protobuf/cmd/protoc-gen-go
+
+      - name: Install goimports
+        run: |
+          go get -u golang.org/x/tools/cmd/goimports
+
+      - name: Install overalls
+        run: |
+          go get -u github.com/go-playground/overalls
+
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          path: go/src/github.com/openconfig/ygot
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+
+      - name: Generate dependencies
+        run: |
+          cd go/src/github.com/openconfig/ygot
+          make generate
+          go build -v ./...
```

Comment:
was this line supposed to remain here or let "Build packages" handle it?

Reply:
It seems like we need to force a build at this point to ensure that there are some dependencies that are retained across steps. I'm not quite sure why this is right now -- I can have a bit of a play around with it and see whether it was something else.

Reply:
This actually looks to have been a bad path -- fixed! Thanks for pointing it out again.